packages/react/src/experimental/Navigation/Header/ProductUpdates/index.tsx

```diff
@@ -0,0 +1,329 @@
+  const [featuredUpdate, ...restUpdates] = updates ?? []
+  const invokeGetUpdates = useCallback(async () => {
+    try {
+      setState("fetching")
+      const response = await getUpdates()
+      setState("idle")
+      setUpdates(response)
+    } catch {
+      setState("error")
+    }
+  }, [getUpdates])
+
+  return (
+    <DropdownMenu
+      onOpenChange={async (open) => {
+        if (open && updates === null) {
+          invokeGetUpdates()
+        }
+      }}
+    >
+      <DropdownMenuTrigger asChild>
+        <button
+          title={label}
+          className="relative inline-flex aspect-square h-8 items-center justify-center rounded border border-solid border-f1-border bg-f1-background-inverse-secondary px-0 text-f1-foreground hover:border-f1-border-hover"
+        >
+          <Icon icon={Megaphone} size="md" />
+          {hasUnread && <UnreadDot className="absolute -right-0.5 -top-0.5" />}
+        </button>
+      </DropdownMenuTrigger>
+      <DropdownMenuPortal>
+        <DropdownMenuContent
+          sideOffset={8}
+          collisionPadding={20}
+          align="end"
+          hideWhenDetached
+          className="max-h-[600px] min-w-96 max-w-md overflow-y-scroll"
+        >
+          <Header title={label} url={updatesPageUrl} />
+          {state === "fetching" && <ProductUpdatesSkeleton />}
+          {state === "idle" && updates !== null && updates.length === 0 && (
+            <>
+              <div className="p-2 pt-0">
+                <NoUpdates {...emptyScreen} buttonUrl={updatesPageUrl} />
+              </div>
+            </>
+          )}
+          {state === "idle" && updates !== null && updates.length > 0 && (
+            <>
+              <FeaturedDropdownItem {...featuredUpdate} />
+              {updates.length > 1 && (
+                <>
+                  <DropdownMenuSeparator />
+                  <p className="text-balance pb-1 pl-3 pr-5 pt-3 text-sm font-medium text-f1-foreground-secondary">
+                    {moreUpdatesLabel}
+                  </p>
+                  {restUpdates.map((update, index) => (
+                    <DropdownItem key={index} {...update} />
+                  ))}
+                </>
+              )}
+            </>
+          )}
+          {state === "error" && (
+            <>
+              <div className="p-2 pt-0">
+                <ErrorScreen
+                  {...errorScreen}
+                  onClick={() => {
+                    invokeGetUpdates()
+                  }}
+                />
+              </div>
+            </>
+          )}
+        </DropdownMenuContent>
+      </DropdownMenuPortal>
+    </DropdownMenu>
+  )
+}
+
+const FeaturedDropdownItem = ({
+  title,
+  href,
+  imageURL,
+  unread,
+  updated,
+}: ProductUpdate) => {
+  const itemClass = "flex flex-col items-stretch gap-3 w-full"
+  return (
+    <DropdownMenuItem asChild>
+      <Link
+        href={href}
```

Comment:
I think forcing it to be an external URL might be a bit too much. The `Link` element is clever enough to understand if this is a relative path or an absolute path. If it's relative, it uses React Router. If not, it sets `target="_blank"`. You can find this on the FactorialOneProvider implementation on the monorepo.

Reply:
what would be your solution? just pass an `href`

Reply:
Oh! What I mean is that we're forcing the type to start with `http://` or `https://` so we won't ever allow internal links, if we want. We could just pass the `href` as a `string` or maybe consider including `/{string}` as a valid type?